src/cow-react/modules/swap/pure/EthFlow/EthFlowStepper/steps/Step1.tsx

```diff
@@ -2,13 +2,15 @@ import React from 'react'
 import Send from 'assets/cow-swap/send.svg'
 import Exclamation from 'assets/cow-swap/exclamation.svg'
 import Checkmark from 'assets/cow-swap/checkmark.svg'
+import X from 'assets/cow-swap/x.svg'
 import { EthFlowStepperProps, SmartOrderStatus } from '..'
 import { StatusIconState } from '../StatusIcon'
 import { Step, ExplorerLinkStyled } from '../Step'
 
 export function Step1({ nativeTokenSymbol, order }: EthFlowStepperProps) {
   const { state, isExpired, createOrderTx } = order
   const isCreating = state === SmartOrderStatus.CREATING
+  const isInvalid = state === SmartOrderStatus.INVALID
 
   let label: string, stepState: StatusIconState, icon: string
```

Comment:
I thought we said that we won't use `let` anymore.
But I'm fine with this.

Reply:
Well, you complain with Anxo who first wrote this :D

I could refactor it but out of scope of the current PR.

What we can do is to make sure we don't as much as possible and remember to point this out when introduced

Reply:
Yep, my fault. and yep, not the end of the world in my opinion this has literally 10 lines of code, and TS ensures that if branches set a value to the 3 variables. 

I think I avoid using it for the others, like `Progress2`. Agree with trying to avoid it